tests/votinghistory.py

```diff
@@ -70,8 +70,8 @@ def run(args):
         for cert, member_id in members_table.items():
             members[member_id] = cert
 
-        if "ccf.voting_history" in tables:
-            votinghistory_table = tables["ccf.voting_history"]
+        if "ccf.governance_history" in tables:
+            votinghistory_table = tables["ccf.governance_history"]
```

Comment:
This test should be updated to confirm that we store the signatures for the proposal as well as the votes.

Reply:
(Actually this is non-critical, feel free to leave for a later PR).